mino/minogrpc/example_test.go

```diff
@@ -11,6 +14,110 @@ import (
 	"go.dedis.ch/dela/serde"
 )
 
+func TestRpcStreamTree(t *testing.T) {
+	NB_NODES := 200
```

Comment:
Go convention is to [not use underscores](https://go.dev/doc/effective_go#mixed-caps). Should be rather `NumNodes` or something like that.